code/modules/surgery/organs/augments_arms.dm

```diff
@@ -0,0 +1,182 @@
+/obj/item/organ/internal/cyberimp/arm
+	name = "arm-mounted implant"
+	desc = "You shouldn't see this! Adminhelp and report this as an issue on github!"
+	zone = "r_arm"
+	slot = "r_arm_device"
+	icon_state = "implant-toolkit"
+	w_class = 3
+	actions_types = list(/datum/action/item_action/organ_action/toggle)
+
+	var/list/items_list = list()
+	// Used to store a list of all items inside, for multi-item implants.
+	// I would use contents, but they shuffle on every activation/deactivation leading to interface inconsistencies.
+
+	var/obj/item/holder = null
+	// You can use this var for item path, it would be converted into an item on New()
+
+/obj/item/organ/internal/cyberimp/arm/New()
+	..()
+	if(ispath(holder))
+		holder = new holder(src)
+
+	update_icon()
+	slot = zone + "_device"
+	items_list = contents.Copy()
+
+/obj/item/organ/internal/cyberimp/arm/update_icon()
+	if(zone == "r_arm")
+		transform = null
+	else // Mirroring the icon
+		transform = matrix(-1, 0, 0, 0, 1, 0)
+
+/obj/item/organ/internal/cyberimp/arm/examine(mob/user)
+	..()
+	user << "<span class='info'>[src] is assembled in the [zone == "r_arm" ? "right" : "left"] arm configuration. You can use a screwdriver to reassemble it.</span>"
+
+/obj/item/organ/internal/cyberimp/arm/attackby(obj/item/weapon/W, mob/user, params)
+	..()
+	if(istype(W, /obj/item/weapon/screwdriver))
+		if(zone == "r_arm")
+			zone = "l_arm"
+		else
+			zone = "r_arm"
+		slot = zone + "_device"
+		user << "<span class='notice'>You modify [src] to be installed on the [zone == "r_arm" ? "right" : "left"] arm.</span>"
```

Comment:
this is the most unintuitive part of all, you should maybe add a line to the description of the implants stating they can be modified with a screwdriver so there's some kind of indication that this can be done